platform/testing/end-to-end/cypress-best-practices-on-vsp.md

```diff
@@ -425,7 +534,7 @@ it.only('should render in mobile layouts and tabs actions work', () => {
       `% traffic for the month of ${percentTrafficPeriod}: ${percentTraffic}%`
     );
 
-    cy.visit('/find-locations');
+    cy.visit("/find-locations");
     cy.injectAxe();
```

Comment:
Does it make sense to use Holden's custom helper that consolidates the injectAxe() and axeCheck() steps?